source/operating-a-service/how-to-categorise-technical-incidents.html.md

```diff
@@ -0,0 +1,100 @@
+# How to categorise technical incidents
+
+The **incident tech lead** needs to triage and categorise the incident to decide on how to respond.
+
+Ask:
+
+* What’s the urgency and why?
+* What’s the impact on our users and services?
+* What’s the extent of the issue and what services and users are affected?
+
+Use the answers to:
+
+1. categorise the incident from the categories below
+2. direct the necessary response.
+
+The categories are fluid and incidents can increase / decrease in priority following further diagnosis or incident resolution work.
+
+## Priority 1 (P1) - High (H)
+
+### Description
+
+Highest and most serious level of incident where **two or more** of the following factors apply:
+
+* 60-100% of users affected.
+* Damage to reputation of service is likely to be high.
+* Support staff are mostly or completely unable to resolve tickets due to service being down.
+
+Examples:
+
+* Total outage of ‘Apply for teacher training’ at any time
+* Total outage of ‘Publish postgraduate teacher training courses’ during ‘peak’ time, which is July-October.
+* Primary search filter on ‘Find postgraduate teacher training courses’ going down. This includes location search and subject selection.
+
+### Stakeholder groups to inform
```

Comment:
I quite like this approach because it means that we then depend on teams having the information of who their SO etc are, in a place they know about and can find. Good exercise to collect that information.